### ISSUE TITLE
examples/Flockers/Readme.md mentions "Flocker_Server.py" - has been renamed or obsoleted?

### ISSUE THREAD

**Issue by danbri** [projectmesa/mesa#476]
Readme says

````
How to Run / Launch the model:  
$ python Flocker_Server.py

Is this handled by run.py now? that does bring up the visualization (and is mentioned later in the readme separately).

**Comment by jackiekazil** [comment]
@danbri thank you for bringing this to our attention. I made some updates which are pending acceptance. You are right in assessment. This ticket will be closed once #477 is merged into master.
